lib/deltaProcessing.js

```diff
@@ -119,6 +119,26 @@ async function dispatch(subjectsWithTypes) {
   for (const individual of subjectsWithTypes) {
     const { subject, type } = individual;
     const organisationUUIDs = await getOrganisationUUIDs(subject, type);
+    const vendor = await getVendorFromSubject(subject);
+    let allowedOrganisationUUIDs;
+    if (vendor) {
+      allowedOrganisationUUIDs = await getAllowedOrganisationUUIDsForVendor(
+        vendor
+      );
+    } else {
+      //No vendor? Then we can't dispatch the data.
```

Comment:
To make sure I understand: suppose the triple with vendor info is not availible yet, what happens in the next cycle? It doesn't look like it's removed from the discard graph or do i miss something?

Reply:
No it is never removed from the discard graph, just like undispatched triples are not removed from the temp-inserts graph. We never know when the rest of the triples for the same set of related subjects will arrive and if we delete some of them too soon, we might not be able to follow a path to the organisation.

Reply:
Ok, but so if the triple comes in later, and previously it was added to the discard graph, it will be correctly dispatched in the next iteration?

Reply:
No, discarded triples will never be dispatched. These triples are published by a vendor that is not allowed to publish information for a certain organisation.

Reply:
but how are we sure all data is available, at the moment this loop runs?

Reply:
We don't. That's why we move it to the discard graph instead of removing it. When can then still use that data to form paths.
(Please keep in mind that this only applies to data that is from a vendor that is not allowed to publish data for the organisation that is found via a path.)